File: Modul6/Python/PRAK603-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
n1, n2 = map(int, input().split())

if n1 == n2:
    matrix1 = list(map(int, input().split()))
    matrix2 = list(map(int, input().split()))

    print()
    
    result_matrix = [matrix1[i] * matrix2[i] for i in range(n1)]
    
    print(" ".join(map(str, result_matrix)))
else:
    print("\nJumlah tidak sama")

File: Modul6/C/PRAK603-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>

int main() {
    int n1, n2;
    
    scanf("%d %d", &n1, &n2);

    int matrix1[n1];
    int matrix2[n2];

    if (n1 == n2) {
        for (int i = 0; i < n1; i++) {
        scanf("%d", &matrix1[i]);
        }
        for (int i = 0; i < n2; i++) {
        scanf("%d", &matrix2[i]);
        }printf("\n");
        for (int i = 0; i < n1; i++) {
            printf("%d ", matrix1[i] * matrix2[i]);
        }  
    } else {
        printf("\nJumlah tidak sama");
    }

    return 0;
}
